### DISCUSSION TITLE
Robot behavior is not updating

### DISCUSSION THREAD

**Discussion by sachinkum0009** [cyberbotics/wrestling#72] [General]
Hi,
Greetings, 
I would like to ask, is there some issue with my repo or something, I tried to update the code but the robot is still doing the same behavior. I am little confused, what did i do wrong.

Please help. Thanks.

**Comment by omichel** [comment]
I could trigger an evaluation of your robot controller. See the [log](https://github.com/cyberbotics/wrestling/actions/runs/4784199989/jobs/8505383399). Unfortunately, your robot lost the game so, it didn't climb up in the ranking, but everything seems to work smoothly. Can you try again?

**Reply by sachinkum0009** [reply]
Thanks
